Tests/scripts/common.py

```diff
@@ -264,63 +265,65 @@ def person_in_charge(commit):
         pr: The GitHub PR link for the Gitlab commit.
     """
     name = commit.author_name
-    email = commit.author_email
     pr_number = commit.title.split()[-1][2:-1]
     pr = f"https://github.com/demisto/content/pull/{pr_number}"
-    return name, email, pr
+    return name, pr
 
 
-def are_pipelines_in_order_as_commits(commits, current_pipeline_sha, previous_pipeline_sha):
-    """
-    This function checks if the commit that triggered the current pipeline was pushed
-    after the commit that triggered the the previous pipeline,
-    to avoid rare condition that pipelines are not in the same order as their commits.
-    Args:
-        commits: list of gitlab commits
-        current_pipeline_sha: string
-        previous_pipeline_sha: string
+def are_pipelines_in_order(current_pipeline, previous_pipeline):
+    previous_pipeline_timestamp = parser.parse(previous_pipeline.created_at)
+    current_pipeline_timestamp = parser.parse(current_pipeline.created_at)
+    return current_pipeline_timestamp > previous_pipeline_timestamp
 
-    Returns:
-        boolean , the commit that triggered the current pipeline
```

Comment:
Why are we removing the docs? If its too much we can replace in a one line.

Reply:
My mistake.